tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -14,16 +14,21 @@ limitations under the License.
 ==============================================================================*/
 
 #include "tensorflow/core/platform/cloud/retrying_file_system.h"
+#include <chrono>
 #include <functional>
+#include <random>
+#include <thread>
 #include "tensorflow/core/lib/core/errors.h"
 #include "tensorflow/core/platform/file_system.h"
 
 namespace tensorflow {
 
 namespace {
 
-// In case of failure, every call will be retried kMaxAttempts-1 times.
-constexpr int kMaxAttempts = 4;
+// In case of failure, every call will be retried kMaxRetries times.
+constexpr int kMaxRetries = 3;
+// Maximum backoff time in seconds.
+constexpr int kMaximumBackoffMilliSeconds = 32000;
```

Comment:
A minor thing, but "millisecond" is one word, so it should be kMaximumBackoffMilliseconds (small "s")